src/base/utils/misc.cpp

```diff
@@ -625,8 +626,15 @@ void Utils::Misc::openFolderSelect(const QString &absolutePath)
     if ((output == "dolphin.desktop") || (output == "org.kde.dolphin.desktop"))
         proc.startDetached("dolphin", QStringList() << "--select" << Utils::Fs::toNativePath(path));
     else if ((output == "nautilus.desktop") || (output == "org.gnome.Nautilus.desktop")
-             || (output == "nautilus-folder-handler.desktop"))
-        proc.startDetached("nautilus", QStringList() << "--no-desktop" << Utils::Fs::toNativePath(path));
+             || (output == "nautilus-folder-handler.desktop")) {
+        proc.start("nautilus", QStringList() << "--version"); //fixes #8923
+        proc.waitForFinished();
+        QString nautilusV = proc.readLine().simplified();
```

Comment:
`const QString nautilusVer`

Reply:
proc.out:         "GNOME nautilus 3.28.1"
Version input: "x.y.z"

Reply:
it do't work, because ''proc.readLine()'' is type = "QByteArray"